resources/virt_test.go

```diff
@@ -46,3 +46,25 @@ func TestExpandHome(t *testing.T) {
 		}
 	}
 }
+
+func TestNumToAlpha(t *testing.T) {
+	var numToAlphaTests = []struct {
+		number int
+		result string
+	}{
+		{0, "a"},
+		{25, "z"},
+		{26, "aa"},
+		{27, "ab"},
+		{702, "aaa"},
+		{703, "aab"},
+		{63269, "cool"},
+	}
+
+	for _, test := range numToAlphaTests {
+		actual := numToAlpha(test.number)
+		if actual != test.result {
+			t.Errorf("numToAlpha(%s): number %s, result %s", test.number, test.result, actual)
```

Comment:
go vet found one small issue:

```
resources/virt_test.go:67: arg test.number for printf verb %s of wrong type: int
```

int's need a `%d` not a `%s`.

Easy to fix... git add -p, git commit --amend, git push .... -f

Reply:
ok. That should do it. Thanks for the heads up.